### PR TITLE
4657 4 calculate penalties

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: web-client/src/presenter/actions/calculatePenaltiesAction.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * Calculates penalties from the current calculate penalties modal
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.get the cerebral get function to retrieve state values
+ * @returns {number} total computed value from penalty values
+ */
+export const calculatePenaltiesAction = ({ get }) => {
+  const { penalties } = get(state.modal);
+
+  const parseCurrency = value => `$${Number(value).toFixed(2)}`;
```
Comment (matthopson) [inline web-client/src/presenter/actions/calculatePenaltiesAction.js:13]: @rachaelparris I like what you're doing in #5221 for this, so I'll refactor to this later.

File: web-client/src/presenter/actions/setTotalPenaltiesAmountForStatisticAction.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * @param {object} providers.store there cerebral store object
+ * @returns {void} sets the form value for the given props.statisticIndex
+ */
+export const setTotalPenaltiesAmountForStatisticAction = ({
+  get,
+  props,
+  store,
+}) => {
+  const { totalPenalties } = props;
+  const { statisticIndex } = get(state.modal);
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/setTotalPenaltiesAmountForStatisticAction.js:18]: It is safer to use `get(state.modal.statisticIndex)` which will never produce an error like "cannot access property statisticIndex of undefined"

Comment (matthopson) [reply web-client/src/presenter/actions/setTotalPenaltiesAmountForStatisticAction.js:18]: Would it ever not be defined?
https://github.com/flexion/ef-cms/blob/develop/web-client/src/presenter/state.js#L204

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/actions/setTotalPenaltiesAmountForStatisticAction.js (hunk 2 above)
Comment (kkoskelin) [reply web-client/src/presenter/actions/setTotalPenaltiesAmountForStatisticAction.js:18]: In this particular case, perhaps not (unless some other action clobbers the entire property).  The basic idea, though, is one I would advocate in general when trying to get values out of the state. It is more 'null-safe' to specify the longest path possible as an argument to `get`.

Comment (matthopson) [reply web-client/src/presenter/actions/setTotalPenaltiesAmountForStatisticAction.js:18]: Yeah. Maybe we could throw a devex task in for this? There's about 90 instances of this in our code, if it's something we should consider addressing. I'm cool with it either way.

**Pull request merged**